ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/container.html

```diff
@@ -42,4 +42,5 @@
         <sly data-sly-use="${'removeattribute.js' @ referencedPage }"/>
     </div>
 </form>
+<div data-cmp-loader="${container.id}"></div>
```

Comment:
Please rename the data attribute to `data-cmp-adaptiveform-container-loader`, also update the component read me